autofaiss/external/optimize.py

```diff
@@ -488,7 +488,7 @@ def optimize_and_measure_indices(
     fs, _ = fsspec.core.url_to_fs(indices_folder)
     indices_file_paths = fs.ls(indices_folder, detail=False)
     index_path2_metric_infos: Dict[str, Dict] = {}
-    suffix_width = int(math.log(len(indices_file_paths))) + 2
+    suffix_width = int(math.log(len(indices_file_paths))) + 1
```

Comment:
Sorry I guess I wasn't clear
In python math.log is the base e logarithm math.log(x) = log(x)/log(e)
What you want here is the base 10 logarithm ie the other function math.log10(x) = log(x) / log(10)

Reply:
It is my bad, you were super clear. Actually I have done the test somewhere using log10 but forgot to copy to the code.

Reply:
Forced pushing a new commit to fix this.